ehitchhiking-api/src/main/java/com/exadel/ehitchhiking/controllers/TripPassengerController.java

```diff
@@ -132,19 +132,19 @@ public Response getAllAvailableTrips(@RequestBody RequestTripPassenger tripPasse
             tripDriverVOList = tripDriverService.getAll(tripPassenger.getEmpId(), tripPassenger.getStartingTime(), tripPassenger.getEndingTime(), tripPassenger.getSeats(),
                     tripPassenger.getCoordStart(), tripPassenger.getCoordEnd());
         } catch (Exception e) {
-            return Response.setError("error");
+            return Response.setError("An error has occurred while retrieving all trips!");
         }
-        return Response.setSuccess(tripDriverVOList, "Successfully got all driver trips");
+        return Response.setSuccess(tripDriverVOList, "All trips were successfully retrieved!");
     }
 
     @PutMapping("/active")
     public Response addToActive(@RequestBody RequestTripPassenger tripPass) {
         try {
             tripDriverService.updateActive(tripPass.getId(), true);
         } catch (Exception e) {
-            return Response.setError("error");
+            return Response.setError("An error has occurred while adding the trip to active!");
         }
-        return Response.setSuccess("true", "Successfully added to active");
+        return Response.setSuccess("true", "The trip was successfully added to active!");
```

Comment:
What?

Reply:
I changed it to "The trip was successfully activated!"